### DISCUSSION TITLE
Error validation

### DISCUSSION THREAD

**Discussion by Abhishek-Sathe-Tafe** [ZiyangWang001007081/Tafe230243#2] [Q&A] [answered]
Are we required to include error validation on the feature branch? For e.g. validating if valid numeric positive integers are entered in for the mortgage calculator?

**Comment by ZiyangWang001007081** [comment]
Hi Abhi,

I agree with including error handling in our code. I would also suggest include a validation mechanism for empty input.

Cheers,
Ziyang

**Comment by Abhishek-Sathe-Tafe** [comment, marked as answer]
Sounds great! I will have that added in. Thanks!
